### ISSUE TITLE
Kernel doesn't get past samsung boot screen 

### ISSUE THREAD

**Issue by gsstudios** [gsstudios/android_kernel_samsung_smdk4412#1]
This is ramdisk related. Since I don't know how to debug very well in kernels, maybe @Lysergic-Acid could help. 

Thanks

**Comment by Lysergic-Acid** [comment]
Can you get me a kmsg?

**Comment by gsstudios** [comment]
Here are some logs attached below
[logs.zip](https://github.com/gsstudios/android_kernel_samsung_smdk4412/files/70430/logs.zip)

**Comment by Lysergic-Acid** [comment]
Weird, the kernel seems to be booting fine there. Are you sure the problem is not in the ROM side? Maybe check the logcat

**Comment by gsstudios** [comment]
Yeah, that's what I also thought. But I start to think that I am missing something and it is causing the kernel to not work properly. 

I'll try building cleanly off gustavo's source and report back.

**Comment by gsstudios** [comment]
Haven't had the time to compile any kernels lately. I might abandon this kernel and continue dorimanx. Isorec may give me something to work with to get this kernel working :)
